packages/fast-components-react-base/src/progress/README.md

```diff
@@ -0,0 +1,5 @@
+# Progress
+The *Progress* component should be used for an element that displays the progress status for a task that take a long time or consists of several steps. Typical use cases include loading elements or status of a download. The *Progress* component is an [div](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div) element with it `role` set to "progressbar".

```

Comment:
please lowercase component names

Reply:
"..for a task that takes a long time..."

Reply:
I was using Button's read me as guidance. We should file a bug to fix the capitalization there.